manual/manual/refman/lex.etex

```diff
@@ -151,13 +151,21 @@ string-literal:
 string-character:
           regular-string-char
         | escape-sequence
+        | "\u{" ("0"\ldots"9"||"A"\ldots"F"||"a"\ldots"f")+ "}"
         | '\' newline { space || tab }
 \end{syntax}
 
 String literals are delimited by @'"'@ (double quote) characters.
 The two double quotes enclose a sequence of either characters
 different from @'"'@ and @'\'@, or escape sequences from the
-table given above for character literals.
+table given above for character literals, or an Unicode character
+escape sequence.
+
+An Unicode character escape sequence is substituted by the UTF-8
```

Comment:
I was always tormented by that. Now I think I eventually understood the rule. The rule is not about the vowel as a letter but as a sound and apparently `y` is not a vowel in english (e.g. a university, not an university). So I think you are right, I'd still like an native english speaker to confirm.